exercises/series/example.lua

```diff
@@ -0,0 +1,7 @@
+return function(s, length)
+  return coroutine.wrap(function()
+    for i = 1, #s - length + 1 do
+      coroutine.yield(s:sub(i, i + length - 1))
+    end
+  end)
+end
```

Comment:
You could do this without coroutines, but this makes it more performant I assume because of the coroutine/iterator pattern?